tests/test_notebooks.py

```
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+"""
+Test all notebooks by executing them with nbconvert.
+"""
+
+import subprocess
+import glob
+from typing import Any
```

Ruff (0.12.2)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)